README.md

```diff
@@ -31,65 +33,9 @@ and follow the steps to "Install SENAITE"
 user/password `admin/admin`
 
 
-## 3. Run OpenMRS EIP sender application on port 8086
-
-#### Set the application-sender.properties
-```
-git clone https://github.com/openmrs/openmrs-eip/
-cd openmrs-eip/
-cp distribution/springboot_setup/sender/_application.properties app/src/main/resources/application-sender.properties
-```
-```
-nano app/src/main/resources/application-sender.properties
-```
-```
-server.port=8086
-
-db-event.destinations=outbound-lims
-
-camel.springboot.xmlRoutes=file:./distribution/springboot_setup/sender/routes/*.xml
-
-debezium.db.user=root
-debezium.db.password=mysql_root_password
-
-debezium.offsetFilename=offset.debezium
-debezium.historyFilename=history.debezium
-
-senaite.baseUrl=http://localhost:8085/senaite
-senaite.username=admin
-senaite.password=admin
+## 3. How to use the routes
 
-openmrs.username=admin
-openmrs.password=your_admin_password
+Follow the instructions here: https://github.com/mekomsolutions/eip-client
 
-spring.openmrs-datasource.username=root
-spring.openmrs-datasource.password=mysql_root_password
-```
-
-#### Optional: Send the log to console
-```
-nano app/src/main/resources/logback.xml
-```
-Add the following log appender
-```
-<appender name="console" class="ch.qos.logback.core.ConsoleAppender">
-  <encoder>
-      <pattern>%-5level %date [%thread] %logger{8}:%line %msg%n</pattern>
-  </encoder>
-</appender>
-<root level="info">
-    <appender-ref ref="console" />
-</root>
-```
-
-#### Run the sender app
-```
-mvn clean install
-java -jar -Dspring.profiles.active=sender app/target/openmrs-eip-app-1.0-SNAPSHOT.jar
-
-```
-## 4. Add SENAITE routes to it
-
-```
-rsync -av routes/ ../openmrs-eip/distribution/springboot_setup/sender/routes/
-```
+#### Note
+Remember to provide an `application.properties` file defining the required runtime properties needed by the routes. Use this [template file](src/test/resources/application.properties). Uncomment the properties prefixed with `##` and set them appropriately.
```

Comment:
Personally I tend to put file names in bold and pieces of code or variable names in code formatting.
I see that in EIP Client the "application.properties" file name is in bold. Let's be consistent and do that here too.

Unless there is an hyperlink, in which case it doesn't need to be bold.

Reply:
Yes, 'template file' is simply a hyperlink to the file.